src/main/java/kostka/moviecatalog/service/StatisticService.java

```diff
@@ -18,13 +18,14 @@
     public static final String ELASTIC_SYNCED_COUNTER = "elasticSyncedCounter";
     public static final String SCHEDULED_SYNCED_COUNTER = "scheduledSyncedCounter";
     public static final String DB_SYNCED_COUNTER = "dbSyncedCounter";
+    public static final int INITIAL_CAPACITY = 5;
```

Comment:
Asi věc stylu. Ale tohle bych jako konstantu možná nedával. Když se na to člověk mrkne, vidí initial capacity... nevíš k čemu to patří. Musíš si dohledat kde se to používá. Klidně bych inlinoval, ale nechám na tobě.

Reply:
klidně